### PR TITLE
Task/dr 798 pipeline infra cleardown

### PR DESCRIPTION
**Pull request by stuartreid**
Cleardown workflow for pipeline-infrastructure-cleardown.dev in service-management repo.

https://nhsd-jira.digital.nhs.uk/browse/DR-798

## Type of changes

- [X] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [X] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [X] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [X] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by gdcNHS — DISMISSED** [review]

File: application/common/utilities.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import os
```
Comment (timrickwood) [inline application/common/utilities.py:1]: I don't think we need this file do we ? I think this is a temp file copied here from utils when we run the build . Possibly a hangover if you ran code locally . I'd remove

File: infrastructure/stacks/service_management [hunk 2]
```diff
@@ -0,0 +1,13 @@
+terraform {
```
Comment (timrickwood) [inline infrastructure/stacks/service_management:1]: I don't think we need this file do we ? I think this is a temp file copied here from utils when we run the build . Possibly a hangover if you ran code locally . I'd remove

File: application/common/common_functions.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
+# import os
```
Comment (timrickwood) [inline application/common/common_functions.py:1]: I don't think we need this file do we ? I think this is a temp file copied here from utils when we run the build . Possibly a hangover if you ran code locally . I'd remove

File: .github/workflows/pipeline-deploy-app-develop.yaml [hunk 4]
```diff
@@ -39,7 +39,7 @@ jobs:
           quality_check,
           metadata,
         ]
-      uses: NHSDigital/uec-dos-management/.github/workflows/deploy-infrastructure.yaml@v0.9
+      uses: NHSDigital/uec-dos-management/.github/workflows/deploy-infrastructure.yaml@v0.11
```
Comment (timrickwood) [inline .github/workflows/pipeline-deploy-app-develop.yaml:42]: You may need to rebase from main to pick up change to remove deploy-support-tools-infrastructure job

**Review by timrickwood — APPROVED** [review]

**Review by Pira-nhs — APPROVED** [review]

**Pull request merged**